src/framework/theme/components/toggle/_toggle.component.theme.scss

```diff
@@ -0,0 +1,103 @@
+/**
+ * @license
+ * Copyright Akveo. All Rights Reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+@mixin nb-toggle-theme() {
+  nb-toggle {
+    margin: nb-theme(toggle-outline-width) 0;
```

Comment:
Why should toggle have outside margin? And if it should, please add specific theme property

Reply:
The component has box-shadow. Margin is necessary to prevent overlap other elements

Reply:
Let's remove margin as discussed irl